dbus-codegen/src/main.rs

```diff
@@ -47,6 +47,8 @@ Defaults to 'RefClosure'."))
              .help("If present, skips a specific prefix for interface names, e g 'org.freedesktop.DBus.'."))
         .arg(clap::Arg::with_name("futures").short("f").long("futures")
              .help("Generates code to use with futures 0.3 (experimental)"))
+        .arg(clap::Arg::with_name("file").long("file").takes_value(true).value_name("FILE")
```

Comment:
I think we want an argument, not an option here. i-e `dbus-codegen-rust [OPTIONS] [INTERFACE_XML_FILE]`. See [here](https://github.com/zeenix/gps-share/blob/master/src/cmdline_config.rs#L32) for an example how that's achieved (the device arg there is an argument, not an option).

Reply:
I kind of prefer keeping it as an option, unless there is more to it than just a matter of taste?

Reply:
Two reasons for arg:

1. I think that's the most typical way to provide input to DBus code generation tool.
2. It's consistent with existing tools. See gdbus-codegen and qdbusxml2cpp for example.